usaspending_api/api_contracts/contracts/v2/disaster/cfda/spending.md

```diff
@@ -37,24 +37,22 @@ Returns spending details of CFDA receiving supplemental funding budgetary resour
             {
                 "results": [
                     {
-                        "id": "43",
-                        "code": "090",
-                        "description": "Description text",
-                        "children": [],
-                        "count": 54,
-                        "obligation": 89.01,
-                        "outlay": 70.98,
-                        "resource_link": "https://beta.sam.gov/fal/25b529f3b5f94b6c939bc0ae8424ae6c/view"
+                        "code": "20.200",
+                        "count": 1,
+                        "description": "CFDA 2",
+                        "id": 200,
+                        "obligation": 20.0,
+                        "outlay": 0.0,
+                        "resource_link": "www.example.com/200"
                     },
                     {
-                        "id": "41",
-                        "code": "012",
-                        "description": "Description text",
-                        "children": [],
-                        "count": 2,
-                        "obligation": 50,
-                        "outlay": 10,
-                        "resource_link": null
+                        "code": "10.100",
+                        "count": 1,
+                        "description": "CFDA 1",
+                        "id": 100,
+                        "obligation": 2.0,
+                        "outlay": 0.0,
+                        "resource_link": None
```

Comment:
```suggestion
                        "resource_link": null
```

Reply:
That is what I get for copy-paste Python test case results. Fixing it now